MeshLib/MeshQuality/MeshValidation.cpp

```diff
@@ -126,6 +126,9 @@ MeshValidation::ElementErrorCodeOutput(const std::vector<ElementErrorCode> &erro
 
 unsigned MeshValidation::detectHoles(MeshLib::Mesh const& mesh)
 {
+	if (mesh.getDimension() == 1)
```

Comment:
Does this fix https://github.com/ufz/ogs/issues/735?
If so, please add a comment about fixing this.